### ISSUE TITLE
[Adaptive Card iOS - InfoClickable.json]: Smily icon below the Clickable icons label is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#171]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Elements' control and activate it.
5. Navigate to 'Infoclickable.json' under 'Elements' list item and activate it.
6. Card will be appear.
7. Perform right swipe and navigate to the card and observe the issue.

**Actual Result:**
While navigating the InfoClickable card smiley under clickable icons label is not accessible through the swipe gesture. 

**Observation:**
This issue is working fine when user touch on the smiley.

**Expected Result:**
Smiley icon under the Clickable icon label should be accessible through the swipe gesture.

**User Impact:**
It will impact the screen reader user as they will not be able to access the smiley icon due to this information which is contains by it may be miss by them.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/a3eb2151-3c6b-417a-88e2-b81c22b5f936

### Card JSON

```JSON
NA
```
